index.html

```diff
@@ -69,11 +71,13 @@ <h2 class="connect-link"> <a href="#">LET'S CONNECT</a></h2>
         <div><img src="images/multi-post-portfolio.png" alt="multi-post-portfolio"></div>
         <div class="tonic-heading">Multi-Post Stories</div>
```

Comment:
- [ ] As required by the previous reviewer, use a header tag instead of a div tag here e.g `<h2>`